{"name": "Unsuccessfully login with email: simeon.hhl.qa@gmail.co and password: Test#1234", "status": "passed", "steps": [{"name": "Check for error", "status": "passed", "start": 1730448538025, "stop": 1730448539765}, {"name": "Check the error text", "status": "passed", "start": 1730448539765, "stop": 1730448539781}], "parameters": [{"name": "email", "value": "'simeon.hhl.qa@gmail.co'"}, {"name": "password", "value": "'Test#1234'"}, {"name": "error", "value": "'There was a problem logging you in. Please try again.'"}, {"name": "driver", "value": "'edge'"}], "start": 1730448537207, "stop": 1730448539781, "uuid": "9e909519-8303-4438-8696-34dca2aed3df", "historyId": "2ffef19ed6000db1495424120c5ac1a1", "testCaseId": "c2811b1721d8231bbb0fb40778ae099c", "fullName": "Tests.test_Login.TestLogin#test_2", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "tag", "value": "flaky(reruns=3, reruns_delay=0.5, rerun_except='assert')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az1776-477"}, {"name": "thread", "value": "1814-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}